doc/book/in-depth-guide/models-and-servicemanager.md

```diff
@@ -596,9 +595,9 @@ template to display the data.
 When pushing variables to the view, they are accessible in two ways: either
 using object notation (`$this->posts`) or implicitly as script-level variables
 `$posts`). The two approaches are equivalent; however, calling `$posts` results
-in a little round-trip through the renderer's `__get()` method. We often
-recommend using `$this` notation to visually differentiate between variables
-passed to the view, and those created within the script itself.
+in a little round-trip through the renderer's `__get()` method <<what renderer? Hasn’t been mentioned yet. Also, doesn’t seem much point in mentioning this unless you explain why it matters.>>. We often
```

Comment:
We're in the "in-depth" guide at this point; the assumption is the user has done the other tutorials, and knows that the renderer is.

The main reason to talk about `__get()` here is so that the user understands that these are not _defined_ properties of the renderer, but instead _proxy properties_. This will explain why their IDE will not provide any hints for the variables.

Reply:
Reference knowing what the renderer is, I would question that the user knows this. 'Rendering' is covered briefly in the Forms and Actions module of the Getting Started module, but the PHPRenderer isn't discussed. If someone is in a similar situation to me (good grounding in OO PHP, but a junior developer and new to ZF and frameworks) and has just done the Getting Started tutorial, they won't know what it is. I would add a link to the relevant documentation both here and in the Getting Started tutorial, i.e. https://docs.zendframework.com/zend-view/php-renderer/ (I've added a link in my patch).

Your comment about `__get` kind of gets (no pun intended!) speaks to a point I made in another patch about what level the tutorial is pitched at. At the moment I think it's pitched a bit too high for the average junior developer / framework newbie (such as myself). To cater for all experience levels, I think it would be helpful to include links to clear explanations of topics when they're brought up for the first time and/or when knowledge is assumed. I've discovered that the ZF reference documentation covers a lot of these topics really well, but the information is buried away in the components library. If links were strategically provided to the relevant ZF docs topic, I think this would make the tutorial more newbie-friendly without breaking the flow and making it too long. As an example I've included such a link in my proposed edit on aliasing a service.